app/api/signup/route.ts

```diff
@@ -1,27 +1,28 @@
+import { DfNicknameValidationUsecase } from "@/application/usecases/auth/DfNicknameValidation";
 import { NextApiRequest, NextApiResponse } from "next";
-import { PrAuthRepository } from "@/infrastructure/repositories/PrAuthRepository";
 import { CheckEmailDuplicationUseCase } from "@/application/usecases/auth/DfCheckEmailDuplicationUsecase";
-import { CheckNicknameDuplicationUseCase } from "@/application/usecases/auth/DfCheckNicknameDuplicationUsecase";
+import { ValidationUUseCase } from "@/application/usecases/auth/DfNicknameValidation";
 import { HashPasswordUseCase } from "@/application/usecases/auth/DfHashPasswordUsecase";
 import { RegisterUserUseCase } from "@/application/usecases/auth/DfRegisterUsecase";
+import { PrUserRepository } from "@/infrastructure/repositories/PrUserRepository";
 
-const authRepository = new PrAuthRepository();
+const userRepository = new PrUserRepository();
```

Comment:
```suggestion
const userRepository: UserRepository = new PrUserRepository();
```
여기도 바꿔주세여 그리고 인스턴스 생성은 함수 안에서 하는 거로 변경해주세욧!!